src/ApparentHorizons/ObjectLabel.hpp

```diff
@@ -0,0 +1,16 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <iosfwd>
+#include <string>
+
+namespace ah {
+/// Labels for the objects in a binary system.
+enum class ObjectLabel { A, B };
```

Comment:
Not a change request, just passing along info for those working on the AH stuff. @wthrowe and @knelli2 your thoughts on this would be appreciated :) I might have asked in the past and forgotten your replies, sorry!

We've had several people complain about `A` and `B` since it's completely unclear how this relates to each object (is A more massive?). The best suggestion we've had is `Primary` and `Secondary`/`Companion` where the `Primary` is the more massive object. There's still an ambiguity with equal-mass, but I think it's better to have only an ambiguity with equal mass rather than every simulation. This probably would require changing a number of places if desired.

Reply:
My current understanding is that object `A` is on the negative x-axis and object `B` is on the positive x-axis. However, there is no convention (that I know of) for which is more massive. If we switch to `Primary`/`Secondary`, the ambiguity basically gets reversed. Now we know which is more massive, but don't know where they are.

The ambiguity in each of these naming schemes is easily solved by just stating that either 
1. object `A` is more massive and on the negative x-axis or
2. `Primary` is more massive and on the negative x-axis

The advantage to using 2 is that the name immediately implies one bit of information (more massive) while 1 doesn't. I think in the long term 2 is better, but this requires a lot of changes so it should be discussed further at a meeting.

Reply:
In SpEC, object A is always on the *positive* x-axis, and is always more (or equally) massive.